db/memorystore.go

```diff
@@ -206,3 +239,29 @@ func (m *MemoryStore) RevokeCertificate(cert *core.Certificate) {
 	defer m.Unlock()
 	delete(m.certificatesByID, cert.ID)
 }
+
+/*
+ * keyToID produces a string with the hex representation of the SHA256 digest
+ * over a provided public key. We use this for acme.Account ID values
```

Comment:
I don't think this comment is accurate anymore w.r.t using this for the acme Account ID, right? Now it's only being used to ensure that duplicate keys aren't used across accounts.

Reply:
Yes, it is. I've updated it.